src/lib/farming/defaultContracts.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * The default farming contract information when a farming contract has never been active.
+ */
+const defaultFarmingContracts = {
+	contractStatus: false as boolean,
+	contractType: '' as '' | 'easy' | 'medium' | 'hard' | '',
+	plantToGrow: '' as string,
+	plantTier: 0 as 0 | 1 | 2 | 3 | 4 | 5,
+	seedPackTier: 0 as 0 | 1 | 2 | 3 | 4 | 5,
+	contractsCompleted: 0 as number
+};
```

Comment:
1. This whole object should probably be typed?
2. Never use `''` as a default value like this, use `null` always to signify that theres no value - this applies to everywhere in this PR.
3. `0 as number` or `false as boolean` really isnt needed. this whole object should be typed to avoid this issue.

Reply:
The typecasting in general is really bad here. We can discuss in discord how to fix it if you want.